src/templates/react/Content/src/State.fs

```diff
@@ -0,0 +1,52 @@
+module State.App
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Import.Browser
+open Global
+open Types.App
+
+let pageParser: Parser<Page->Page,Page> =
+  oneOf [
+    map About (s "about")
+    map Counter (s "counter")
+    map Home (s "home")
+  ]
+
+let urlUpdate (result: Option<Page>) model =
+  match result with
+  | None ->
+    console.error("Error parsing url")
+    ( model,Navigation.modifyUrl (toHash model.currentPage) )
+  | Some page ->
+      match page with
+      | Counter ->
+          let (counter, counterCmd) = State.Counter.init ()
```

Comment:
@et1975 
How would you handle application which have an initial model state and an initial pool of `Cmd` ?
For example, your sub application need to fetch the list of the users from the server.

The solution could be something like:

```fs
      let initialCmd =
        match page with
        | Counter -> Cmd.map CounterMsg (State.Counter.initCmd ())
        | _ -> []
      { model with currentPage = page }, initialCmd
```